include/revng/Model/schema.yml

```diff
@@ -0,0 +1,701 @@
+preamble: |
+  #pragma once
+
+  #include <compare>
+  #include <optional>
+
+  #include "llvm/ADT/StringRef.h"
+  #include "llvm/Support/YAMLTraits.h"
+  #include "revng/ADT/RecursiveCoroutine.h"
+  #include "revng/Model/Identifier.h"
+  #include "revng/Model/TupleTree.h"
+  #include "revng/Model/VerifyHelper.h"
+  #include "revng/Support/MetaAddress.h"
+
```

Comment:
This YAML file should strive to be language agnostic. One day we could drop `SortedVector` in favor of some other syntax.

For sure, no includes.
These can be pushed in a `.h` file somewhere.

Why do you need to include `VerifyHelper.h` and ` RecursiveCoroutine.h` everywhere?
The others seem fine.

Reply:
There is no need to include `VerifyHelper.h` and `RecursiveCoroutine.h` anymore (it used to be because I used to declare `verify` in the autogenerated classes). I removed them.
I'm not taking action re: the other includes, because they might become redundant once we move the YAML definitions to their corresponding files (we could use the same includes found in the file or maybe no includes at all as all the necessary headers would be included by the manually written file)